.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 
-python: 3.5
+python: 3.8
 env:
-    - TOXENV=py27
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py36
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
     - TOXENV=coverage
 before_install:
```

Comment:
I did some googling, and while I did not find the exact reason why above code no longer works, I found a hint how to fix it.

Instead of using the `env` keyword, it looks like nowadays you have to use a `matrix` as follows:
```
language: python

matrix:
    include:
        - python: "3.6"
          env: TOXENV=py36
        - python: "3.7"
          env: TOXENV=py37
        - python: "3.8"
          env: TOXENV=py38
        - python: "3.8"
          env: TOXENV=pep8
        - python: "3.8"
          env: TOXENV=coverage

before_install:
```

If you replace these lines it should work.

Reply:
Ok done! I was also wondering why this code fails to pass checks